src/api/app/controllers/webui/request_controller.rb

```diff
@@ -445,15 +470,26 @@ def change_state(newstate, params)
   def accept_request
     flash[:success] = "Request #{params[:number]} accepted"
 
-    # Check if we have to forward this request to other projects / packages
+    # Forward the requests when using the beta request show page
```

Comment:
You really need to rename this method to `forward_requests` or something. It is super confusing...

Reply:
Done